air-quality-backend/database_tests/utils/CAMS_forecast_tests.py

```diff
@@ -0,0 +1,46 @@
+from database_tests.utils.db_helpers import get_database_data
+from dotenv import load_dotenv
+
+load_dotenv()
+
+collection_name = "forecast_data"
+
+
+def test_overall_aqi_level_is_between_1_and_6():
```

Comment:
My opinion, which could be wrong here, is that we don't need to test every single document in the database for these test cases. My issues with it are around the volume of data we will test against (i.e. we're going to end up with around 12000 documents a day) as well as I'm not sure it's really necessary if we unit test the function that calculates AQI value anyway.

If I understand correctly, this code is going to be tweaked and run manually every now and then only against QA database which will continually be cleaned down.

But if this is intended to be automated I think we need another approach. For example if we really need to test if aqi values are in a certain range we can make that query in mongo itself and assert a count of 0 for example

Reply:
Aye, this is true. Just have them in there for ease of access 😎